src/main/io/serial_4way.c

```diff
@@ -422,6 +422,11 @@ void esc4wayProcess(serialPort_t *mspPort)
     uint8_t *InBuff;
     ioMem_t ioMem;
 
+    if (!mspPort)
+    {
```

Comment:
I was attempting to be consistent with the rest of the code below... would you like me to re-commit?